tests/TotalPriceCalculation.spec.ts

```diff
@@ -0,0 +1,41 @@
+import {test, expect} from '@playwright/test'
+import {NavigationPage } from '../page-objects/navigationPage'
+import {LightingPage } from '../page-objects/lightingPage'
+
+
+test.beforeEach(async ({page})=>{
+    
+    await page.goto('https://www.shopist.io/')
+})
+
+
+test('navigate to chairs page', async ({page})=>{
```

Comment:
What is this test used for? What are we assert? Did you manage ti run this test successfully?